projects/common/src/time/time-range.service.ts

```diff
@@ -64,14 +64,23 @@ export class TimeRangeService {
   }
 
   private initializeTimeRange(): void {
-    this.navigationService.navigation$
+    combineLatest([
```

Comment:
this.navigationService.navigation$
.pipe(
   switchMap(activatedRoute => ([activatedRoute, activatedRoute.queryParamMap]))
)

Reply:
please explain the logic in a comment

Reply:
This syntax doesn't work because it doesn't 'unwrap' the queryParamMap from it's observable, so it ends up return an array with items of type `ActivatedRoute | Observable<ParamMap>`

`Type 'ActivatedRoute | Observable ' must have a '[Symbol.iterator]()' method that returns an iterator`

Reply:
aah yea. i guess the combine latest is fine for now